extensions/oidc-token-propagation-common/runtime/src/main/java/io/quarkus/oidc/token/propagation/common/AccessToken.java

```diff
@@ -1,4 +1,4 @@
-package io.quarkus.oidc.token.propagation;
+package io.quarkus.oidc.token.propagation.common;
```

Comment:
I wonder if I would move it to a `io.quarkus.oidc.token.propagation.annotations` package instead.

We had quite a lot of complaints of people having to add a lot of Quarkus libs to libraries they share and I /think/ that if we actually move annotations to fix issues, we should isolate them in a package so that we could imagine splitting them to a small runtime artifact later (not a full extension, just a small runtime artifact depending on nothing). It would makes sense only if we do 

@dmlloyd WDYT, am I being too cautious?

Reply:
I don't really care about `io.quarkus.oidc.token.propagation.common` vs `io.quarkus.oidc.token.propagation.common.annotations`, so I'll let @sberyozkin answer. I
d just note that this annotation is useless in a standalone library, the annotation is just something we can easily find with Jandex in a build step and once we extract information (annotation target, attribute values) we don't care about it.

Users are not going to work with this new unlisted extension as it is included already. Was your point about that if the annotation in `io.quarkus.oidc.token.propagation.annotations`, it could be easily separated from other Java classes that we would add to the API - `io.quarkus.oidc.token.propagation.common`  (which is empty ATM BTW) ? I just want to understand.

Reply:
Yeah so I know nothing about `@AccessToken` and I'm not sure it makes sense in this case so let's take an example for which it makes sense so that's it's easier to grasp :).

`@RegisterForReflection` is in `quarkus-core`. So if you have a common library (of entities or CDI beans) that is shared between let's say your old services using WildFly and your new services using Quarkus, you drag a whole bunch of Quarkus stuff into the WildFly server (except if you had proper exclusions).
You could also have a shared library of JAX-RS resources for instance.

We had several complaints about that along the years thus why I thought we should start isolating the annotations more actively - especially when we are moving them.

Now maybe it makes no sense at all for this particular case.

Reply:
> I wonder if I would move it to a `io.quarkus.oidc.token.propagation.annotations` package instead.
> 
> 
> 
> We had quite a lot of complaints of people having to add a lot of Quarkus libs to libraries they share and I /think/ that if we actually move annotations to fix issues, we should isolate them in a package so that we could imagine splitting them to a small runtime artifact later (not a full extension, just a small runtime artifact depending on nothing). It would makes sense only if we do 
> 
> 
> 
> @dmlloyd WDYT, am I being too cautious?

It seems reasonable to me.